### PR TITLE
At post-install, ask the user to turn on command line autocompletion

### PR DESCRIPTION
**Pull request by teobugslayer**
Implements https://github.com/NativeScript/nativescript-cli/issues/350

### CODE REVIEW

File: commands/completion-prompt.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+				private $logger: ILogger,
+				private $prompter: IPrompter) {
+	}
+
+	public disableAnalytics = true;
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$logger.out("If you are using bash or zsh, you can enable command line auto completion.");
```
Comment (rosen-vladimirov) [inline commands/completion-prompt.ts]: Maybe we should notify the user that we will write some data in bash_profile, zsh and bashrc

**Comment by rosen-vladimirov** [comment]
:+1:  after considering my suggestion

File: commands/completion-prompt.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			var message = "Enable it now?";
+
+			var autoCompetionStatus = this.$prompter.confirm(message, () => "y").wait();
```
Comment (Fatme) [inline commands/completion-prompt.ts]: Shouldn't we prompt this message only when we have iteractive console?

File: commands/completion-prompt.ts (hunk 1 above)
Comment (teobugslayer) [inline commands/completion-prompt.ts]: Done - in the help file

File: commands/completion-prompt.ts (hunk 2 above)
Comment (teobugslayer) [inline commands/completion-prompt.ts]: Will restore the code (it used to be there)

**Comment by Fatme** [comment]
:+1:

**Pull request merged**